docs/howto/import.md

```diff
@@ -22,6 +22,22 @@ To avoid copying the data, lakeFS offers [Zero-copy import](#zero-copy-import).
 
 ### Prerequisites
 
+#### User Permissions
+
+The lakeFS user running the import must have the following permissions:
+`fs:WriteObject`, `fs:CreateMetaRange`, `fs:CreateCommit` and 
+`fs:ImportFromStorage`. The first 3 permissions are available by default to 
+users in the default Developers group ([RBAC](../reference/rbac.md)) or the 
+Writers group ([ACL](../reference/access-control-lists.md)). The last 
+permission enables the user to import data from any location of the storage 
+provider that lakeFS has access to. Thus, it's only available to users in the 
```

Comment:
Maybe we should emphasize this sentence. Or even add it as some kind of warning on top

Reply:
IMO it's bad enough that we have to start an how-to page with permissions adjustments for some non-trivial use-cases. Adding warnings on top of that gets them further from the value..